api/WorkerLocation.json

```diff
@@ -513,22 +513,22 @@
               "version_added": null
             },
             "opera": {
-              "version_added": true
+              "version_added": "≤15"
```

Comment:
Why a range here?

Reply:
Admittedly, the collector doesn't actually test `toString`.  The Safari and other Chromium data was mirrored from Chrome Desktop.  I hadn't tested this function in Opera Presto, and since Opera doesn't contribute to our current real values goal, I didn't dig into this further.

Reply:
I see, that's fine. There aren't many "≤14" or "≤15" left and I've been keen to remove it as an allowed range, but fine to keep using it to not get bogged down with testing Presto in our current effort.